shared/ci/config/nodejs-dependabot.yml

```
@@ -8,7 +8,7 @@
 
 version: 2
 updates:
-- package-ecosystem: "npm" # See documentation for possible values
+- package-ecosystem: "pnpm" # See documentation for possible values
```

```suggestion
- package-ecosystem: "npm" # See documentation for possible values
```

see: https://github.com/blinkbitcoin/mavapay-client/runs/39437670342